pom.xml

```diff
@@ -301,8 +301,34 @@
         </executions>
       </plugin>
 
-      <!-- We use this assembly to package the GWT RPC dependencies so they 
-				are aviablible fore integration testing. -->
+      <plugin>
+        <groupId>com.fizzed</groupId>
+        <artifactId>fizzed-watcher-maven-plugin</artifactId>
+        <version>1.0.6</version>
+        <!-- This configuration is used to watch the node_modules directory for changes and copy those to the static assets folder when they change -->
+        <!-- Then those assets are copied into the exploded WAR -->
+        <configuration>
+          <touchFile>target/classes/watcher_explode_war.txt</touchFile>
+          <watches>
+            <watch>
+              <directory>
+                ${project.basedir}/node_modules/synapse-react-client/dist/umd
+              </directory>
```

Comment:
Watch synapse-react-client for changes. We can update this list as needed; I didn't want to start with watching the entire node_modules folder (not sure the performance implications of watching 80k+ files